05-lists-and-dictionaries/capital_city_loop.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from capitals import capitals_dict
+from random import choice
+
+def get_random_state():
+    return choice(list(capitals_dict.keys()))
+
+def capital_of_state_quiz(state):
+    while True:
+        response = input("What is the capital of " + state + "? ").lower()
+
+        if check_answer(response, capitals_dict[state]):
+            print("Correct")
+            return True
+        elif response == "exit":
+            print("The correct answer is: " + capitals_dict[state])
+            print("Goodbye")
+            return True
+        else:
+            print("Wrong - try again or type exit to quit")
+            return capital_of_state_quiz(state)
```

Comment:
Either use `while True` or recursive calls, not both (they do the same thing and we don't need both). I personally prefer the `while True`.